src/sing/audioRendering.ts

```diff
@@ -744,7 +744,9 @@ class SynthVoice {
       stopContextTime < this.stopContextTime
     ) {
       // リリースのスケジュールを行う
-      this.gainNode.gain.cancelAndHoldAtTime(t0);
+
+      // cancelAndHoldAtTimeはFirefoxでサポートされていないので、?.()で呼び出す
+      this.gainNode.gain.cancelAndHoldAtTime?.(t0);
```

Comment:
コメントに２行使わなくても良さそうなのでちょっと変えさせていただきます！
```suggestion
      this.gainNode.gain.cancelAndHoldAtTime?.(t0); // Fiefoxで未対応
```